### PR TITLE
Update README 

### PR DESCRIPTION
**Pull request by fkiraly**
Suggested updates to the GitHub landing page:

* updated abstract; refers to community activities too
* index at the start of the page
* tabular overview of supported learning tasks and non-estimator object; this links to relevant artefacts such as tutorials, documentation, etc - I wanted to make this emojis, but we can't make emojis in `rst`?
* clear vision statement
* extended new contributors section to be more clear on how to contact us or to contribute

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -52,17 +52,34 @@
...
+* `Time series classification/regression <https://github.com/alan-turing-institute/sktime/blob/main/examples/02_classification_univariate.ipynb>`__,
+
+including feature extraction, pipelining, tuning, and model benchmarking functionality.
+
+Support for additional learning tasks is under development and/or experimental:
+
+* time series clustering
+* anomaly detection, change-point detection
+* outlier detection & removal, segmentation
+* segmentation (supervised and unsupervised, stream and panel)
```
Comment (mloning) [inline README.rst:80]: segmentation is already mentioned above
```suggestion
* multivariate forecasting
```

File: README.rst [hunk 2]
```diff
@@ -52,17 +52,34 @@
 Welcome to sktime
 =================
 
-  A unified framework for machine learning with time series
+The unified framework for machine learning with time series. Our vision:
```
Comment (mloning) [inline README.rst:55]: I still prefer "A" - don't think we need to claim being the "special" one in whatever sense that may be.

File: README.rst [hunk 3]
```diff
@@ -52,17 +52,34 @@
...
 
-We provide specialized time series algorithms and `scikit-learn
-<https://github.com/scikit-learn/scikit-learn>`__ compatible tools
-to build, tune and validate time series models for multiple learning problems, including:
+* **by the community**, for the community. Developed by a friendly, decentral community - `we welcome all contributors <https://gitter.im/sktime/community>`__.
```
Comment (mloning) [inline README.rst:57]: To me, having the vision at the top of the readme is a bit long, it also reads a bit too opinionated for a landing page I think. 

As an example, I really like spaces readme: https://github.com/explosion/spaCy/blob/master/README.md Short description, then well-organised overview tables with important links, then short summaries of features, installation, etc.

Comment (mloning) [reply README.rst:57]: I prefer the vision statement to go into the community roadmap

File: README.rst [hunk 4]
```diff
@@ -167,15 +184,18 @@ We follow the `all-contributors specification <https://github.com/alan-turing-in
 
 If you have a question, `chat <https://gitter.im/sktime/community?source=orgpage>`__ with us or `raise an issue <https://github.com/alan-turing-institute/sktime/issues/new/choose>`__. Your help and feedback is extremely welcome!
 
+All new contributors are welcome. You can join to contribute by:
```
Comment (mloning) [inline README.rst:187]: This section now reads a bit strange, I like your list of entry points below for how to get involved, perhaps organising them in a table would be good. 

For example:
| How to get involed | |
|---|---|
| [Mentoring](...) | Do you want to contribute to sktime but need some help? |
| [Meet-ups](...) | ... |

**Comment by mloning** [comment]
Having said this, I agree, the landing page needs to be updated, but I'd try to follow spacy's readme as a good and engaging example.

File: README.rst (hunk 2 above)
Comment (Lovkush-A) [reply README.rst:55]: I have preference for 'The'. But it is not strong preference. Either would be fine

**Comment by fkiraly** [comment]
hm, do you want to give it a second go, @mloning?

File: README.rst (hunk 2 above)
Comment (TonyBagnall) [reply README.rst:55]: I prefer "A", "The" seems a bit too much to claim

Comment (fkiraly) [reply README.rst:55]: ok, fine with me

Comment (fkiraly) [reply README.rst:55]: what about "the scikit-learn-like"? that is more like a definition

File: README.rst (hunk 3 above)
Comment (fkiraly) [reply README.rst:57]: hm, I prefer it here - @TonyBagnall, what do you think?

**Comment by fkiraly** [comment]
All, I've given this another re-work, please review.

I've addressed @mloning's comments by moving vision further down; at the top, there is now an abstract and an index.
Further down there is a table of features.

**Comment by mloning** [comment]
I've re-written the REAME file (changed to `README.md` now). 

* inspired by https://github.com/explosion/spaCy 
* kept most of your additions @fkiraly but restructured/reformatted them 
* checked PyPI compliance/uploading via `make clean && make dist && twine check dist/*`
* sktime.org homepage is no longer linked to README, markdown conversion wasn't very stable, README and home page (index.rst) also serve different purpose, I suggest we re-write index.rst during the doc sprint

**Review by mloning — DISMISSED** [review]
Reviewed @fkiraly's and my own changes.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
+<a href="https://sktime.org"><img src="https://github.com/alan-turing-institute/sktime/blob/main/docs/source/images/sktime-logo-no-text.jpg?raw=true)" width="175" align="right" /></a>
+
+# Welcome to sktime
+
+> A unified interface for machine learning with time series
+
+:rocket: **Version 0.7.0 out now!** [Check out the release notes here.](https://github.com/alan-turing-institute/sktime/releases)
+
+sktime is a library for time series analyis in Python. It provides a unified interface for multiple time series learning tasks, including time series classification, regression, clustering, annotation and forecasting. It comes with time series algorithms and [scikit-learn] compatible tools to build, tune and validate time series models.
```
Comment (GuzalBulatova) [inline README.md:9]: Typo in "for time series analyis" - `analySis`.
Are there other ML tasks that can be solved with our tools, except the ones counted here? If there aren't then we could just say "It provides a unified interface for time series classification, regression, clustering, annotation and forecasting."

Comment (mloning) [reply README.md:9]: Not yet, but hopefully in the future, I'll update it to say that this is what we currently support - thanks!

File: README.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+[github discussions]: https://github.com/alan-turing-institute/sktime/discussions
+[stack overflow]: https://stackoverflow.com/questions/tagged/sktime
+[gitter]: https://gitter.im/sktime/community
+[discord]: https://discord.com/invite/gqSab2K
+
+
+## :stars: Features
+sktime provides a __unified interface for distinct but related time series learning tasks__. It features __dedicated time series algorithms__ and __tools for composite model building__ including pipelining, ensembling, tuning and reduction that enables users to apply an algorithm for one task to another.
+
+sktime also provides **interfaces to related libraries**, for example [scikit-learn], [statsmodels], [tsfresh], [PyOD] and [fbprophet], among others. Our aim is to make the time series analysis ecosystem more interoperable and usable as a whole.
```
Comment (GuzalBulatova) [inline README.md:62]: I think it makes sense to have `Our aim is to make the time series analysis ecosystem more interoperable and usable as a whole.` as an opening statement in the beginning of `Features`.

**Comment by fkiraly** [comment]
Obviously I can't self-approve because this is originally my PR, but I'm happy with this and would approve if I could, @mloning.
I thought the "modules" table was nice and would have deserved more columns (even if not at the top), but we can fight over minor details later.

**Pull request merged**